evm/src/main/java/org/hyperledger/besu/evm/precompile/KZGPointEvalPrecompiledContract.java

```diff
@@ -134,8 +135,16 @@ public PrecompileContractResult computePrecompile(
                 Optional.of(ExceptionalHaltReason.PRECOMPILE_ERROR));
       }
       return result;
+    } catch (CKZGException ckzgException) {
```

Comment:
This seems like debug code that should be reverted right? also remove the System.out.println